hadoop-hdds/docs/content/integration/Impala.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Impala
+weight: 4
+menu:
+   main:
+      parent: "Application Integrations"
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Overview
+
+Starting with version 4.2.0, Apache Impala provides full support for querying data stored in Apache Ozone. To utilize this functionality, ensure that your Ozone version is 1.4.0 or later.
+
+# Supported Access Protocols
+
+Impala supports the following protocols for accessing Ozone data:
+
+* ofs
+* s3a
+
+Note: The o3fs protocol is **NOT** supported by Impala.
+
+# Supported Replication Types
+
+Impala is compatible with Ozone buckets configured with either:
+
+* RATIS (Replication)
+* Erasure Coding
+
+# Querying Ozone Data with Impala
+
+Impala provides two approaches to interact with Ozone:
+
+* Managed Tables
+* External Tables
+
+## Managed Tables
+
+If the Hive Warehouse Directory is located in Ozone, you can execute Impala queries without any changes, treating the Ozone file system like HDFS. For example:
+
+```sql
+  CREATE DATABASE d1;
```

Comment:
The `CREATE EXTERNAL TABLE` example is not indented, so I think the others can be unindented as well.

```suggestion
CREATE DATABASE d1;
```